tools/nut-scanner/nut-scanner.c

```diff
@@ -210,7 +219,9 @@ void show_usage()
 	}
 
 	printf("  -E, --eaton_serial <serial ports list>: Scan serial Eaton devices (XCP, SHUT and Q1).\n");
-
+#ifdef HAVE_PTHREAD
+        printf("  -T, --thread <max number of threads>: max number of simultaneaous threads (default %d). \n", DEFAULT_THREAD);
```

Comment:
Indentation is different from neighboring code.
Trailing space after period in the help text.

Reply:
Also a typo in "simultaneaous"